src/adservice/build.gradle

```diff
@@ -88,13 +99,6 @@ task downloadRepos(type: Copy) {
     into offlineCompile
 }
 
-task adService(type: CreateStartScripts) {
-    mainClassName = 'hipstershop.AdService'
-    applicationName = 'AdService'
-    outputDir = new File(project.buildDir, 'tmp')
-    classpath = startScripts.classpath
-}
-
```

Comment:
Why is this removed?

Reply:
It wreaks havoc with the gradle plugin that expects to install the java agent via "normal" means. The previous version of the gradle build script had the topmost start scripts disabled and then re-patched in via this custom mechanism. Gradle doesn't do well with overcustomization. :)

Reply:
Ok, sounds good. Can we remove the usage instead of commenting it then?